metrics/Inactive_Contributers.md

```diff
@@ -0,0 +1,33 @@
+# Inactive_Contributers
+
+Question: How many Contributers have gone inactive over a specific period of time.
+
+## Description
+A metric that shows how many contributers have stopped committing over a specific period of time. The period of time required for a contributer to be counted as inactive can be decided by a variable and this metric will display the number of contributers that have been labeled as inactive over a given time frame.
+
+## Objectives
+The objective is to determine how many people are losing intrest in the project. This can be useful in determining intrest in the project. User intrest and user retention is an important metric for determing the health of a project and watching how many users are going inactive can help in finding those values. It is important to know if many people are leaving the project and have stopped commiting.
+
+## Implementation
+The metric will take in 2 variables a period and a cutoff. The period will be the period over which the number of inactive members will be displayed. For example if period=year then it will display the number of contributers that have gone inactive each year. The cutoff will determine how long it takes for a contributer to be labeled as inactive. If a previous contributer has not made any commits for longer than the cutoff, then they will be added to the inactive count of the period of their last commit. Because of this there will be a bit of a delay between when a user stops commiting and a user shows up as inactive thus this metric will not work for days more recent than the cuttoff.
```

Comment:
Change: 
"If a previous contributer has not made any commits for longer than the cutoff, then they will be added to the inactive count of the period of their last commit. Because of this there will be a bit of a delay between when a user stops commiting and a user shows up as inactive thus this metric will not work for days more recent than the cuttoff."

If a contributor has not made a contribution for a length of time longer than the cutoff, they will be counted as inactive. 

Change "period" to "time period", which is the range of dates we are looking for inactive contributors. Change "cuttoff" to "interval" for the length of time a person will be inactive before we count them as inactive.

Reply:
your definition of period is not what I originally intended. I intended it to be more of a graph period. As in if period is a year it would show how many went inactive in each year. Did you still want me to change it? I hadn't really thought too deeply into it as it was just a rough draft so I don't mind the change in definition.

Reply:
I see what you are saying. What we really mean is "what is the number of days back" you want look altogether. So, for example, the default "time period" could be "all time", or it could be 3 years, for example. And the "interval" that determines inactivity could be 4 months without any commits or issues.  

I think the original may be a little too complex to understand, and the simpler case we propose will be immediately useful for community managers.

Reply:
@sgoggins Thanks for the clarification. It was a little confusing but its clearer now.

Signed-off-By: Armstrong<foundjem@ieee.org>